rconweb/api/urls.py

```diff
@@ -14,51 +20,102 @@
     vips,
     votemap,
 )
+from .auth import api_response, login_required
+
+
+def _get_empty(value):
+    """Return None for inspect._empty values or the value as a string"""
+    if value is _empty:
+        return None
+    else:
+        return str(value)
+
+
+@csrf_exempt
+@login_required()
```

Comment:
does this really need a login, btw.? 🤔

Reply:
Probably not now that I think about it, we already expose the CRCON version publicly even when someone isn't logged in, and if you know the version then you know the API.

The only time it wouldn't match is if someone was building their own images on an altered build.

I'll remove the permissions.